docs/src/@primer/gatsby-theme-doctocat/nav.yml

```diff
@@ -110,3 +110,5 @@
       url: /components/alpha/underlinenav
     - title: UnderlinePanels
       url: /components/alpha/underlinepanels
+    - title: Layout - Fix my order in docs/src/@primer/gatsby-theme-doctocat/nav.yml
```

Comment:
Does this need to be addressed?

Reply:
Looks like it was a duplicate entry. Removed here: https://github.com/primer/view_components/pull/889/commits/02ae8af072c81513015c608ded005a7de761648b

Reply:
Looks like it was a duplicate entry. Removed here: https://github.com/primer/view_components/pull/889/commits/02ae8af072c81513015c608ded005a7de761648b